solutions/instances/variables.tf

```
@@ -123,6 +123,12 @@ variable "cloud_monitoring_provision" {
   default     = true
 }
 
+variable "existing_cloud_monitoring_crn" {
+  description = "The CRN of existing IBM Cloud Monitoring instance"
```

```suggestion
  description = "The CRN of an IBM Cloud Monitoring instance."
```

remove `existing`. Can't have a CRN from a non-existing instance. Also added the article `an`

---

addressed